gss/gokrb5.go

```diff
@@ -345,25 +346,39 @@ func loadCache() (*credentials.CCache, error) {
 	return cache, nil
 }
 
+func findFile(env string, try []string) (string, error) {
+	path, ok := os.LookupEnv(env)
+	if ok {
+		if _, err := os.Stat(path); err != nil {
+			return "", err
+		}
+		return path, nil
+	}
+
+	var errs error
```

Comment:
Is it possible/practical to add an error to the multi error if the ENV variable was not found? That would result in something like this:

```
=== RUN   TestAccDnsTXTRecordSet_Basic
    resource_dns_txt_record_set_test.go:47: Step 1/6 error: Error running apply: exit status 1

        Error: Error updating DNS record: Error negotiating GSS context: 2 errors occurred:
        	* KRB5_CONFIG: environment variable not found, checking for default config paths
        	* stat /etc/krb5.conf: no such file or directory



          with dns_txt_record_set.foo,
          on terraform_plugin_test.tf line 3, in resource "dns_txt_record_set" "foo":
           3:   resource "dns_txt_record_set" "foo" {
```

I'm not well versed in how this library is consumed outside of our terraform provider, so nbd if it doesn't make sense

Reply:
I've added a simple `KRB5_CONFIG: not found` error as the first value in the multierror and I've also wrapped any `os.Stat()` error when checking the value of that environment variable so the difference is obvious if someone happened to set `KRB5_CONFIG=/etc/krb5.conf`.